File: Maximum_Strong_Pair_XOR_I.java
import java.util.List;

class Solution {
    public int maximumStrongPairXor(List<Integer> nums) {
        int ans = Integer.MIN_VALUE;
        for (int x : nums) {
            for (int y : nums) {
                if (Math.abs(x - y) <= Math.min(x, y)) {
                    ans = Math.max(ans, x ^ y);
                }
            }
        }
        return ans;
    }
}

File: Maximum_Strong_Pair_XOR_I.cpp
class Solution {
public:
    int maximumStrongPairXor(vector<int>& nums) {
        int ans=INT_MIN;
        for(int x:nums)
        {
            for(int y:nums)
            {
                if(abs(x-y)<=min(x,y))
                {
                    ans=max(ans,x^y);
                }
            }
        }
        return ans;

        
    }
};
